manim/mobject/mobject.py

```diff
@@ -66,6 +82,14 @@ class Mobject(Container):
 
     """
 
+    animation_overrides: Dict[
+        Type["Animation"], Callable[["Mobject"], "Animation"]
+    ] = {}
+
+    def __init_subclass__(cls):
```

Comment:
I know this automatically gets converted to a classmethod by python, but I'd prefer being explicit about it nonetheless. Also best practice is to forward `**kwargs` on to the parent method, even if `object.__init_subclass__` just errors if it gets keyword arguments.
```suggestion
    @classmethod
    def __init_subclass__(cls, **kwargs):
        super().__init_subclass__(**kwargs)

```